### ISSUE TITLE
Shortkey / Hotkey / keybindings

### ISSUE THREAD

**Issue by Theknight2015** [Thinstation/thinstation#857]
Hey all! I just want to say ahead of time, thank you for any help or info you can provide.

I'm sure this issue has been covered somewhere before but I can't seem to find any problems, discussions, issues or questions (open or closed) on any forum regarding my issue specifically.

I would like to know if there is a way to setup key bindings on thinstation so that something like "Ctrl + Alt + T" would open the Terminal or something like that and as of yet I have found nothing and nothing I try works on the thin clients.

The reason I'm asking is because I have the terminal package uncommented in the build.conf file and I have a few custom thinstation.conf-pc-tc-xx files where pc-tc-xx is the host name of the specific thin clients I have setup.

The thinstation.hosts file defines the hostnames in relation to MAC addresses of the devices. This works great for custom configs like being able to have Terminal, Chrome, and Freerdp all installed/uncommented in build.conf but make it so that pc-tc-01 may have all those on the desktop icons and pc-tc-02 may only have a couple Freerdp icons on the desktop.

What I'd like to do (using pc-tc-02 as an example since it only has a couple Freerdp icons) is set it up so when I hit a specific key binding, the terminal application opens. I'm assuming that since pc-tc-02 simply has the icon turned off for the desktop/start menu that the terminal program is still available and it's just a matter of having a way to launch it right?

In simple terms, I want the desktop to still only show the Freerdp icons but when I press the key binding/hotkey combo for terminal, it still opens even though there is no icon for it.

Is this possible? Am I crazy? I thought there was info on this at one point but either I'm not looking in the right spots, it's been removed or maybe I am in fact crazy. Hopefully one of you can tell me.

Thanks again to anyone who can help me!

**Comment by Doncuppjr** [comment]
have a look at build/packages/xfwm4/build/extra/etc/skel/.config/xfce4/xfconf/xfce-perchannel-xml/xfce4-keyboard-shortcuts.xml

**Comment by Theknight2015** [comment]
Perfect, thank you so much! I changed the custom section from this:

`<property name="custom" type="empty">
      <property name="&lt;Control&gt;Escape" type="string" value="xfdesktop --menu"/>
      <property name="&lt;Alt&gt;F2" type="string" value="xfce4-appfinder --collapsed"/>
      <property name="&lt;Alt&gt;F3" type="string" value="xfce4-appfinder"/>
      <property name="&lt;Alt&gt;F1" type="string" value="xfce4-popup-applicationsmenu"/>
      <property name="&lt;Control&gt;&lt;Alt&gt;Delete" type="string" value="xflock4"/>
      <property name="XF86Mail" type="string" value="exo-open --launch MailReader"/>
      <property name="XF86Display" type="string" value="xfce4-display-settings --minimal"/>
      <property name="XF86WWW" type="string" value="exo-open --launch WebBrowser"/>
      <property name="&lt;Super&gt;p" type="string" value="xfce4-display-settings --minimal"/>
      <property name="override" type="bool" value="true"/>
      <property name="Print" type="string" value="mtpaint -s"/>
      <property name="&lt;Super&gt;l" type="string" value="xflock4"/>
</property>
`



To This:
`<property name="custom" type="empty">
      <property name="&lt;Control&gt;Escape" type="string" value="xfdesktop --menu"/>
      <property name="&lt;Alt&gt;F2" type="string" value="xfce4-appfinder --collapsed"/>
      <property name="&lt;Alt&gt;F3" type="string" value="xfce4-appfinder"/>
      <property name="&lt;Alt&gt;F1" type="string" value="xfce4-popup-applicationsmenu"/>
      <property name="&lt;Control&gt;&lt;Alt&gt;Delete" type="string" value="xflock4"/>
      <property name="XF86Mail" type="string" value="exo-open --launch MailReader"/>
      <property name="XF86Display" type="string" value="xfce4-display-settings --minimal"/>
      <property name="XF86WWW" type="string" value="exo-open --launch WebBrowser"/>
      <property name="&lt;Super&gt;p" type="string" value="xfce4-display-settings --minimal"/>
      <property name="override" type="bool" value="true"/>
      <property name="Print" type="string" value="mtpaint -s"/>
      <property name="&lt;Super&gt;l" type="string" value="xflock4"/>
      <property name="&lt;Control&gt;&lt;Alt&gt;t" type="string" value="xfce4-terminal"/>
</property>
`

And now when I click Ctrl + Alt + T it works! I would assume this would also work for other applications as well. I may try that at some point. Thanks again!

P.S. - I'll be making another thread about displays and having multiple displays if I can't figure it out on my own so you may see that in a couple hours as well. Maybe sooner.
